ceph_deploy/util/log.py

```diff
@@ -53,6 +54,7 @@ def format(self, record):
         truncated_level = record.levelname[:6]
         levelname_color = COLOR_SEQ % (30 + COLORS[levelname]) + truncated_level + RESET_SEQ
         record.color_levelname = levelname_color
+        record.timestamp = datetime.datetime.now().strftime('%Y-%m-%d %H:%M:%S.%f')
```

Comment:
Isn't this only for colored output formatting that goes only to the terminal/stderr? I guess it doesn't hurt to leave it there